src/HOHQMesh.jl

```diff
@@ -77,6 +95,9 @@ function generate_mesh(control_file;
     end
   end
 
+  # Append a statement to indicate to the user which mesh file format was used
+  output = string(output, "\n", " Mesh file written in ", mesh_file_format, " format.")
```

Comment:
Do we really want to do this here? I don't like the idea of mixing HM.jl output with the original output of HM itself.

Reply:
I thought it would be nice, but now that I have played with this a few times I don't like it anymore. The user should know what format they set in the control file, so they don't need reminded of it. I will take this out.